conda/recipes/librmm/meta.yaml

```diff
@@ -24,13 +24,12 @@ build:
 
 requirements:
   build:
-    - cmake >=3.12.4
+    - cmake >=3.14.0
   host:
     - cudatoolkit {{ cuda_version }}.*
-    - spdlog >=1.4.2
   run:
     - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
-    - spdlog >=1.4.2
+    - spdlog >=1.6.0
```

Comment:
is `spdlog` actually needed at runtime? Isn't it a header only library that just gets compiled into librmm?

Reply:
Yes if someone includes the rmm header https://github.com/rapidsai/rmm/blob/branch-0.14/include/rmm/mr/device/logging_resource_adaptor.hpp, as we do for the cuDF JNI code. Here "runtime" is a slight abuse of term since it's really the compile time for c++.